install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -0,0 +1,272 @@
+openshift_hostname_check: false
+ansible_become: true
+
+*#Setting SDN (defaults to ovs-networkpolicy) not part of OSEv3.yml*
+#For more info, on which to choose, visit: 
+#https://docs.openshift.com/container-platform/3.10/architecture/networking/sdn.html#overview
+networkPluginName: redhat/ovs-networkpolicy
+#networkPluginName: redhat/ovs-multitenant
+
+*#Configuring identity providers with Ansible*
+#For initial cluster installations, the Deny All identity provider is configured
+#by default. It is recommended to be configured with either htpasswd
+#authentication, LDAP authentication, or Allowing all authentication (not recommended)
+#For more info, visit:
+#https://docs.openshift.com/container-platform/3.10/install_config/configuring_authentication.html#identity-providers-ansible
+#Example of Allowing All
+#openshift_master_identity_providers: [{'name': 'allow_all', 'login': 'true', 'challenge': 'true', 'kind': 'AllowAllPasswordIdentityProvider'}]
+
+
+*#Optional Metrics (uncomment below lines for installation)*
+
+#openshift_metrics_install_metrics: true
+#openshift_metrics_cassandra_storage_type: dynamic
+#openshift_metrics_storage_volume_size: 25Gi
+#openshift_metrics_cassandra_nodeselector: {"node-role.kubernetes.io/infra":"true"}
+#openshift_metrics_hawkular_nodeselector: {"node-role.kubernetes.io/infra":"true"}
+#openshift_metrics_heapster_nodeselector: {"node-role.kubernetes.io/infra":"true"}
+
+*#Optional Aggregated Logging (uncomment below lines for installation)*
+
+#openshift_logging_install_logging: true
+#openshift_logging_es_pvc_dynamic: true
+#openshift_logging_es_pvc_size: 30Gi
+#openshift_logging_es_cluster_size: 3
+#openshift_logging_es_number_of_replicas: 1
+#openshift_logging_es_nodeselector: {"node-role.kubernetes.io/infra":"true"}
+#openshift_logging_kibana_nodeselector: {"node-role.kubernetes.io/infra":"true"}
+#openshift_logging_curator_nodeselector: {"node-role.kubernetes.io/infra":"true"}
+
+----
+
+For further details on any of the variables listed, visit: 
+https://github.com/openshift/openshift-ansible/blob/master/inventory/hosts.example
+
+
+=== OpenStack Prerequisites Playbook
+
+The {product-title} Ansible Installer provides a playbook to ensure all the provisioning
+steps of the OpenStack instances have been met. 
+
+Prior to running the playbook, ensure to source the RC file
+
+----
+$ source path/to/examplerc
+----
+
+Via the `ansible-playbook` command on the deployment host, ensure all the
+prerequisites are met using `prerequisites.yml` playbook:
+
+[subs=+quotes]
+----
+$  ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/openstack/openshift-cluster/prerequisites.yml
+----
+
+Once the prerequisite playbook completes successfully, run the provision playbook
+as follows:
+
+----
+$ ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/openstack/openshift-cluster/provision.yml
+----
+
+[IMPORTANT]
+
+====
+The _provision.yml_ prematurely errors, check if the status of the 
```

Comment:
Do you mean "If provision.yml prematurely errors, check the status of..."?

Reply:
yes i did. will fix.